CMakeLists.txt

```diff
@@ -50,6 +50,17 @@ if (WITH_ROS)
         add_subdirectory(bindings/ros)
 endif()
 
+# install udev rules
```

Comment:
Could group this commands into a cmake file that can be stored in the `cmake` folder, and just include it here with a comment line ######################### Udev Rules ####################### or something to match the rest of the comments in this file

Reply:
solved. thanks